apps/ledger-live-desktop/src/renderer/modals/Platform/Exchange/CompleteExchange/Body.tsx

```diff
@@ -163,24 +170,30 @@ const Body = ({ data, onClose }: { data: Data; onClose?: () => void | undefined
           return onCancel(new DisabledTransactionBroadcastError());
         }
         onResult(operation);
-        // else not swap i.e card and sell we close the drawer
+      } else if (
+        (data.exchangeType === ExchangeType.SELL || data.exchangeType === ExchangeType.SELL_NG) &&
+        sourceCurrency
+      ) {
```

Comment:
Can we check "quoteId" as I am not sure if an old flow us using "ExchangeType.SELL"

```suggestion
const isValidPtxSellFlow= data.exchangeType === ExchangeType.SELL || data.exchangeType === ExchangeType.SELL_NG) &&
        sourceCurrency
        (isValidPtxSellFlow) {
```